python/lsst/ts/standardscripts/maintel/laser_tracker/align.py

```diff
@@ -0,0 +1,206 @@
+# This file is part of ts_standardscripts
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+
+__all__ = ["Align"]
+
+import yaml
+
+from lsst.ts import salobj
+from lsst.ts.idl.enums.LaserTracker import LaserStatus, LaserTrackerComponent
```

Comment:
The `LaserTrackerComponent` will not be available in ts_idl for a while, even if we make the PR now. 

So, for now, you will have to define it here and add a TODO to replace it with the one in ts_idl once that one is available.

Reply:
Done